cmd/file_lint.go

```diff
@@ -0,0 +1,165 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strings"
+
+	"github.com/daveshanley/vacuum/motor"
+	"github.com/daveshanley/vacuum/rulesets"
+	"github.com/kong/go-apiops/filebasics"
+	"github.com/kong/go-apiops/logbasics"
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdLintInputFilename string
```

Comment:
It's not clear what `lint-file` is in the usage, so this usage appears undefined to me.  If we want to allow positional arguments, it seems we need to decide if that is the input state file(s) or the rules file(s).   Can the underlying library support multiple rules files?  It may make sense to allow a single input state file via `-s,--state` and multiple rules files as the positional arguments.

Reply:
> Can the underlying library support multiple rules files?

The library gets a single input, so it's a matter of implementing the logic in decK to merge the rulesets.

It may make sense to just support a single state file and a single ruleset for starters. WDYT?

Reply:
I don’t see a use case for supporting multiple at this point. Users can run the tool multiple times, or use the merge function to generate a single input